### PR TITLE
[tests-only][full-ci]Refactor test function for handeling errors

### PR DESCRIPTION
**Pull request by SwikritiT**
## Description
Follow up of: [tests-only][full-ci]Seperate test steps for positive and negative cases #5733

## Related Issue

- Part of: https://github.com/owncloud/ocis/issues/5646

## How Has This Been Tested?

- locally
- CI 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/acceptance/features/bootstrap/GraphContext.php [hunk 1]
```diff
@@ -1689,11 +1686,8 @@ public function theAdministratorTriesToAddTheFollowingUsersToAGroupAtOnceUsingTh
 		$userIds = [];
 		$groupId = $this->featureContext->getAttributeOfCreatedGroup($group, "id");
 		foreach ($table->getHash() as $row) {
-			try {
-				$userIds[] = $this->featureContext->getAttributeOfCreatedUser($row['username'], "id");
-			} catch (Exception $e) {
-				$userIds[] = WebDavHelper::generateUUIDv4();
-			}
+			$userId = $this->featureContext->getAttributeOfCreatedUser($row['username'], "id");
+			$userIds[] = !$userId ? WebDavHelper::generateUUIDv4() : $userId;
```
Comment (saw-jan) [inline tests/acceptance/features/bootstrap/GraphContext.php:1690]:
```suggestion
			$userIds[] = $userId ? $userId : WebDavHelper::generateUUIDv4();
```

File: tests/acceptance/features/bootstrap/FeatureContext.php [hunk 2]
```diff
@@ -4589,18 +4589,18 @@ public function getUserIdByUserName(string $userName): string {
 				throw new Exception(__METHOD__ . " accounts-list is empty");
 			}
 		}
-		throw new Exception(__METHOD__ . " user with name $userName not found");
+		return false;
```
Comment (saw-jan) [inline tests/acceptance/features/bootstrap/FeatureContext.php:4592]: this should throw
```suggestion
		throw new Exception(__METHOD__ . " user with name $userName not found");
```

File: tests/acceptance/features/bootstrap/FeatureContext.php [hunk 3]
```diff
@@ -4616,6 +4616,6 @@ public function getGroupIdByGroupName(string $groupName): string {
 				throw new Exception(__METHOD__ . " accounts-list is empty");
 			}
 		}
-		throw new Exception(__METHOD__ . " Group with name $groupName not found");
+		return false;
```
Comment (saw-jan) [inline tests/acceptance/features/bootstrap/FeatureContext.php:4619]: this should throw
```suggestion
		throw new Exception(__METHOD__ . " Group with name $groupName not found");
```

File: tests/acceptance/features/bootstrap/GraphContext.php [hunk 4]
```diff
@@ -192,12 +192,8 @@ public function editUserUsingTheGraphApi(string $byUser, string $user, string $u
...
 	public function adminHasRetrievedUserUsingTheGraphApi(string $user): void {
 		$user = $this->featureContext->getActualUsername($user);
-		try {
-			$userId = $this->featureContext->getAttributeOfCreatedUser($user, "id");
-			$userId = $userId ?? $user;
-		} catch (Exception $e) {
-			$userId = $user;
-		}
+		$userId = $this->featureContext->getAttributeOfCreatedUser($user, "id");
+		$userId = !$userId ? $user : $userId;
```
Comment (saw-jan) [inline tests/acceptance/features/bootstrap/GraphContext.php:196]:
```suggestion
		$userId = $userId ? $userId : $user;
```

**Review by saw-jan — APPROVED** [review]
LGTM

**Review by SagarGi — APPROVED** [review]
LGTM :+1:

**Pull request merged**